assets/lokomotive-kubernetes/bootkube/resources/charts/kube-apiserver/templates/kube-apiserver.yaml

```diff
@@ -144,6 +147,12 @@ spec:
         - --requestheader-group-headers=X-Remote-Group
         - --requestheader-username-headers=X-Remote-User
         {{- end }}
+        {{- range .Values.apiserver.extraFlags }}
+        - {{ . }}
+        {{- end }}
+        readinessProbe:
```

Comment:
This shouldn't be here I think.

Reply:
Why ?

Reply:
Because it is not related to the extra kube-apiserver flags. Also, we already have `readinessProbe` defined 10 lines later. I guess this is a leftover after a bad conflict resolving while rebasing.